views/importing/queue_items.html

```
@@ -45,15 +45,60 @@ <h4 class="text-sm font-semibold text-gray-900 dark:text-white truncate">{{.Trac
+         <div class="flex flex-wrap items-center gap-2">
+             <button _="on click toggle .hidden on #player-queue-{{.ID}}"
+                 class="inline-flex items-center gap-1.5 px-2.5 py-1 rounded-md text-xs font-medium bg-violet-500/10 border border-violet-400/30 text-violet-600 dark:text-violet-300 hover:bg-violet-500/20 transition-colors">
+                 <i class="fas fa-music fa-xs"></i> New
+             </button>
+             <button _="on click toggle .hidden on #player-lib-{{.ID}}"
+                 class="inline-flex items-center gap-1.5 px-2.5 py-1 rounded-md text-xs font-medium bg-blue-500/10 border border-blue-400/30 text-blue-600 dark:text-blue-300 hover:bg-blue-500/20 transition-colors">
+                 <i class="fas fa-music fa-xs"></i> Existing
+             </button>
+         </div>
+         <div id="player-queue-{{.ID}}" class="audio-player hidden flex items-center gap-2.5 bg-violet-500/5 border border-violet-400/20 rounded-lg px-3 py-2 w-full">
+             <button onclick="toggleAudioPlayer(this)" class="play-btn flex-shrink-0 w-6 h-6 flex items-center justify-center rounded-full bg-violet-500/20 hover:bg-violet-500/40 text-violet-600 dark:text-violet-300 transition-colors">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)